src/Plot/Scale.elm

```diff
@@ -0,0 +1,32 @@
+module Plot.Scale where
+
+import Private.Scale.Linear as LinearScale
+import Private.Scale.OrdinalBands as OrdinalBands
+import Private.Extras.Set as Set exposing (Set)
+import Private.Scale exposing (Scale)
+
+type alias LinearScale = Scale Set Float
+type alias OrdinalScale = Scale (List String) String
+
+linear : (Float, Float) -> (Float, Float) -> Int -> LinearScale
+linear domain range numTicks =
+  { domain = Set.createFromTuple domain
+  , range = Set.createFromTuple range
+  , interpolate = LinearScale.interpolate
+  , uninterpolate = LinearScale.uninterpolate
```

Comment:
This is the same terminology that D3 uses. I struggled to find something better to describe this mapping. https://github.com/mbostock/d3/blob/master/src/scale/linear.js#L23